ml-notebook/packages.txt

```diff
@@ -209,31 +208,31 @@ icu==69.1
 idna==3.3
 imagecodecs==2022.2.22
 imageio==2.16.1
-importlib-metadata==4.11.2
-importlib_metadata==4.11.2
-importlib_resources==5.4.0
+importlib-metadata==4.11.3
+importlib_metadata==4.11.3
+importlib_resources==5.6.0
 iniconfig==1.1.1
 intake==0.6.5
 intake-esm==2021.8.17
 intake-geopandas==0.4.0
 intake-stac==0.3.0
 intake-xarray==0.6.0
-ipykernel==6.9.1
+ipykernel==6.12.0
 ipyleaflet==0.15.0
 ipyspin==0.1.5
-ipython==8.1.1
+ipython==8.2.0
 ipython_genutils==0.2.0
 ipytree==0.2.1
 ipyurl==0.1.2
-ipywidgets==7.6.5
+ipywidgets==7.7.0
 isodate==0.6.1
 jasper==2.0.33
-jax==0.3.0
+jax==0.3.4
```

Comment:
note bump

Reply:
This looks like https://github.com/conda-forge/jax-feedstock/issues/81